ordered_associative_task/student/include/impl.h

```diff
@@ -15,4 +15,22 @@
  *
  * @return 1-5
  */
-size_t convertMarkTo5PointSystem(size_t markIn100Point);
+std::size_t convertMarkTo5PointSystem(std::size_t markIn100Point)
+{
+    if(markIn100Point > 100)
```

Comment:
Looks hacky, try to solve it with such mapping:
{{0, 1}, {30, 2}, {50, 3}, {70, 4}, {90, 5}};